docs/_articles/en/einstein/einstein-example-prompts.md

```diff
@@ -76,19 +76,24 @@ return orderItems;
 
 ```
 
-**Prompt:** Write an Apex trigger to add the number of contacts associated with an account.
+**Prompt:** Write an Apex trigger to add the number of contacts associated with an account. In the SOQL query ensure it is secure by adding WITH USER_MODE.
 
 ```
-trigger gptcontacttrigger on Account (after insert, after update) {
-    List<Contact> listContacts = new List<Contact>();
-
-    for (Account acc : Trigger.new) {
-        List<Contact> listaccContacts = [SELECT Id, Name, AccountId FROM Contact WHERE AccountId = :acc.Id];
-        if (acc.Contacts.size() < listaccContacts.size()) {
-            acc.addError('Not enough contacts');
-        }
-
+trigger AccountTrigger on Account (after update) {
+    List<Account> accountsWithContacts = [
+        SELECT Id, (SELECT Id FROM Contacts)
+        FROM Account
+        WITH USER_MODE
```

Comment:
@sbudhirajadoc looks like we lose the WHERE clause on this from the prior version, so now it's querying *all* accounts, which doesn't match the prompt. 

As-is in this diff it tries to query ALL accounts and contacts which is just gonna blow out limits, so dangerous example.

Reply:
updated the response with chris's comment.